modules/terraform-aci-macsec-interfaces-policy/main.tf

```diff
@@ -0,0 +1,35 @@
+resource "aci_rest_managed" "macsecIfPol" {
+  dn         = "uni/infra/macsecifp-${var.name}"
+  class_name = "macsecIfPol"
+
+  content = {
+    name    = var.name
+    adminSt = var.admin_state ? "enabled" : "disabled"
+  }
+}
+
+resource "aci_rest_managed" "macsecRsToKeyChainPol" {
+  dn         = "uni/infra/macsecifp-${var.name}/rsToKeyChainPol"
```

Comment:
change to 
dn = "${aci_rest_managed.macsecIfPol.dn}/rsToKeyChainPol
and delete the dependencies